src/app/messages.ts

```diff
@@ -38,12 +60,39 @@ export const errMessages = {
     }
   },
   Login: {
-    
+    "login": "Oops... Invalid email or password, please type it again."
+  },
+  PasswordValidation: {
+    mismatch: {
+      "mismatch": "Password mismatch ..."
+    },
+    minlength: {
+      "minlength": "Password minimum length is 8 characters ..."
+    }
   },
   Registration: {
-
+    mismatch: {
+      "match": "The passwords entered do not match."
+    },
+    error: {
+      "error": "Sorry, something went wrong. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    invalidUser: {
+      "account": "Whoops... Account not found. Please contact " // after the "contact", the content will be a variable, please check with Registration/model.ts file to view detail
+    },
+    noPassword: {
+      "password": "Whoopes... Unable to register, invalid password."
+    },
+    alreadyRegistered: {
+      "registered": "This account has already registered. Please type your registered email and password to login."
+    }
   },
   ResetPassword: {
-
+    resetLoginFailed: {
+      "failed": "Whoops, login failed, please go to login page to sign in."
```

Comment:
resetPassword and resetLoginFailed are a confusing combination for names. I'm not sure what actually failed here.
If it's resetting of the password that failed then I don't think going to the login page to sign in would help.
"Whoops, we were unable to reset your password. Please try again."

If it's trying to reset your password and then log you in but failing on the login part.
"Whoops, were were unable to log you in directly after resetting your password. Please return to the login page and try logging in again."